source/includes/_resource.customers.md

```diff
@@ -132,6 +137,17 @@ vhx.customers.create({
       </td>
       <td>Billing parameters for <a href="/in-app-purchases">In-App Purchases</a> for the Apple, Google, and Roku platforms.</td>
     </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td class="nowrap">
+        <strong class="is-block text--navy">plan</strong>
+        <span class="is-block text--transparent text-3">string</span>
+        <span class="text--transparent text-3">optional</span>
```

Comment:
Do we have some existing examples of noting the "default"? The default for plan is currently "standard" (all API hits to /customer without a plan=free for AVOD is basically thought as a full "paying" subscriber).